app/_redirects

```diff
@@ -165,6 +165,8 @@
 /konnect/manage-plugins/enable-route-plugin/         /konnect/servicehub/enable-route-plugin/
 /konnect/manage-plugins/shared-config                /konnect/runtime-manager/manage-proxy-config/
 /konnect/shared-config                               /konnect/runtime-manager/gateway-config/
+/konnect/vitals/*                                    /konnect/analytics/:splat
+/konnect/vitals/analyze/                             /konnect/analytics/services-and-routes/
```

Comment:
```suggestion
/konnect/vitals/analyze/                             /konnect/analytics/services-and-routes/
/konnect/vitals/*                                    /konnect/analytics/:splat
```
Switching the order. The one that comes first gets resolved first, so the wildcard is messing with the other redirect.